src/fft.js

```diff
@@ -467,9 +467,7 @@ define(function (require) {
    *                               Defaults to 0.8.
    */
   p5.FFT.prototype.smooth = function(s) {
-    if (s) {
-      this.smoothing = s;
-    }
+    this.smoothing = s;
     this.analyser.smoothingTimeConstant = s;
```

Comment:
@TheBrenny ah, sorry, it was confusing that I used `this.fft` in the example above when I should have used `this.analyser`. I'll try to clarify...

First of all, your comment is spot on that 0 is falsey and so your update is definitely an improvement. Anything beyond that is nice to have, let me know if you want to give it a shot.

I'm thinking, what if somebody calls `fft.smooth('some string');` ? Then in the smoothing function we'd see this go through successfully...

```this.smoothing = 'some string'```

...but this throw an error

`this.analyser.smoothingTimeConstant = 'some string'`

Same thing would happen if smoothing is above 1, or below 0.

So I am suggesting to take `this.smoothing` out of [the smooth() method](https://github.com/TheBrenny/p5.js-sound/blob/6de0770487c0dac22f085a4c81de7088122ec207/src/fft.js#L471), and instead define the property as a "getter" in the constructor. Something like this:

```
p5.FFT = function(smoothing, bins) {
    var SMOOTHING = smoothing || 0.8;

    ...
    this.analyser.smoothingTimeConstant = SMOOTHING;

    ...
    Object.defineProperty(this, 'smoothing', {
      get: function() {
        return this.analyser.smoothingTimeConstant;
      }
      set: function(s) {
         this.analyser.smoothingTimeConstant = s;
      }
    });
}
```

If we create an instance called fft, and set `fft.smoothing = 0`, that calls the setter, which does the same thing `fft.smooth(0)` would do. If we `console.log(fft.smoothing)` that will call the getter, returning `this.analyser.smoothingTimeConstant` in real time.

We haven't used getters and setters in p5.sound much, but seems like a useful place to incorporate them.